docs/dashboards/kpi-component.md

```diff
@@ -0,0 +1,96 @@
+KPI Component
+===
+
+The KPIComponent allows you to visualize key performance indicators.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-kpi allow="fullscreen"></iframe>
+
+## How to start
+1. The KPIComponent is included in the Dashboard package, so nothing apart from the basic dashboards package needs to be imported.
```

Comment:
KPI is not included in the main Dashboards package. It is registered in the HighchartsPlugin

Reply:
The bottom line here is that you do not need to include an extra module for the KPI, so it's in the core of the Dashboards.